project/src/components/navigation/navigation.tsx

```diff
@@ -0,0 +1,57 @@
+import { Link } from 'react-router-dom';
+import { connect, ConnectedProps } from 'react-redux';
+import { Dispatch } from 'react';
+
+import { Actions } from '../../types/action';
+import { State } from '../../types/state';
+
+import { setCity } from '../../store/actions';
+import { CITIES } from '../../const';
+
+type PropsFromRedux = ConnectedProps<typeof connector>;
+type ConnectedComponentProps = PropsFromRedux & NavigationCityProps;
+
+type NavigationCityProps = {
+  // city: string,
+}
+
+const mapStateToProps = ({currentCity}: State) => ({
+  currentCity,
+});
+
+const mapDispatchToProps = (dispatch: Dispatch<Actions>) => ({
+  onCityChange(city: string) {
+    dispatch(setCity(city));
+  },
+});
+
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+function Navigation(props: ConnectedComponentProps): JSX.Element {
+  const {onCityChange, currentCity} = props;
+
+  return (
+    <section className="locations container">
+      <ul className="locations__list tabs__list">
+        {Object.values(CITIES)
+          .map((item) => item.name)
+          .map((item) => (
```

Comment:
```suggestion
          .map((item) => (
```
делай все в одном мапе